helm/polaris/templates/deployment.yaml

```diff
@@ -31,6 +31,9 @@ spec:
   {{- if not .Values.autoscaling.enabled }}
   replicas: {{ .Values.replicaCount }}
   {{- end }}
+  {{- if not (has (quote .Values.revisionHistoryLimit) (list "" (quote ""))) }}
```

Comment:
@MonkeyCanCode we just merged support for helm-unit. If you update your branch, maybe you could add a small unit test for this field? Especially for the logic around differentiating `0` from `nil` or `""`. Thanks 🙏

Reply:
Nice. Will look into this later today.

Reply:
@adutra this had being added. I really like this approach with validating helm (compared to how I was doing it earlier via helm template). I can add the remaining test cases in for the next PR if there is no concern on your end and no one is already working on it.

Reply:
Excellent tests 👍 
And OK to add missing test cases little by little.